l10n_ar_website_sale_ux/models/product_template.py

```diff
@@ -0,0 +1,54 @@
+from odoo import models
+
+
+class ProductTemplate(models.Model):
+    _inherit = "product.template"
+    
+    def _get_sales_prices(self, pricelist):
+        template_price_vals = super()._get_sales_prices(pricelist)
```

Comment:
Add docstring with importance of this change, Example:
""" RG 4/2025 force us to show both prices in the e-commerce, with taxes and without taxes. If we have configured a tax included website then we compute the price_tax_excluded to be shown. (this  is shown in the product list/kanban view) """